### ISSUE TITLE
filenames case sensitive on Linux (/Mac)

### ISSUE THREAD

**Issue by webonix** [MadnessLabs/MadnessLife#1]
Hi Bobby

I think a filename change is required
services/Database.ts -> database.ts

> [ ERROR ]  typescript: src/components.d.ts:37:8
>            Cannot find module './services/database'.

**Comment by popcorn245** [comment]
Yeah that is my bad.  Windows doesn't notice a difference so even when I have them changed on my local it doesn't sync the case change.  I will resolve it though because you're awesome @webonix ^_^
